downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -0,0 +1,166 @@
+| *`automationhub_ldap_bind_password`* | _Required_
+
+The password to use with `automationhub_ldap_bind_dn`.
+| *`automationhub_ldap_group_search_base_dn`* | An LDAPSearch object that finds all LDAP groups that users might belong to. 
+If your configuration makes any references to LDAP groups, this and `automationhub_ldap_group_type` must be set.
+
+Default = '`None`'
+| *`automatiohub_ldap_group_search_filter`* | _Optional_
+
+Search filter for finding group membership.
+
+Default = `'(objectClass=Group)'`
+| *`automationhub_ldap_group_search_scope`* | _Optional_
+
+Default = `'SUBTREE'`
+| *`automationhub_ldap_group_type_class`* | _Optional_
+
+Default =`'django_auth_ldap.config:GroupOfNamesType'`
+| *`automationhub_ldap_server_uri`* | The URI of the LDAP server. 
+This can be any URI that is supported by your underlying LDAP libraries.
+| *`automationhub_ldap_user_search_base_dn`* | An LDAPSearch object that locates a user in the directory. 
+The filter parameter should contain the placeholder %(user)s for the username. 
+It must return exactly one result for authentication to succeed.
+| *`automationhub_main_url`* | When using Single Sign-On, specify the main {HubName} URL that
+clients will connect to, e.g. `\https://<hubaddress.example.com>`, which leads to the external address being entered in `/etc/pulp/settings.py`.
+
+If not specified, the first node in the `[automationhub]` group is used.
+| *`automationhub_pg_database`* | _Required_
+
+The database name.
+
+Default = `‘automationhub’`
+| *`automationhub_pg_host`* | Required if not using internal database.
+| *`automationhub_pg_password`* | The password for the {HubName} postgreSQL database.
+
+Do not use special characters for `automationhub_pg_password`. 
+They can cause the password to fail.
+| *`automationhub_pg_port`* | Required if not using internal database.
+
+Default = 5432
+| *`automationhub_pg_sslmode`* | Required.
+
+Default = `‘prefer’`
+| *`automationhub_pg_username`* | Required 
+
+Default = `‘automationhub’`
+| *`automationhub_require_content_approval`* | _Optional_
+
+If {HubName} enforces the approval mechanism before collections are made available.
+
+By default when you upload collections to {HubName} an administrator must approve it before it is made available to the users. 
+
+If you want to disable the content approval flow, the variable should be set to `false`.
+
+Default = `true`
+| *`automationhub_ssl_cert`* | _Optional_ 
+
+`/path/to/automationhub.cert`
+Same as `web_server_ssl_cert` but for {HubName} UI and API
+| *`automationhub_ssl_key`* | _Optional_
+
+`/path/to/automationhub.key`
+
+Same as `web_server_ssl_key` but for {HubName} UI and API
+| *`automationhub_ssl_validate_certs`* | For {PlatformName} 2.2 and later, this value is no longer used.
+
+If {HubName} should validate certificate when requesting itself because by default, Platform deploys with self-signed certificates.
```

Comment:
```suggestion
If {HubName} should validate certificate when requesting itself because by default, {PlatformNameShort} deploys with self-signed certificates.
```